api/utils.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.sia.tech/jape"
+	"go.sia.tech/renterd/internal/prometheus"
+)
+
+func WriteResponse(jc jape.Context, code int, resp any) error {
+	var responseFormat string
+	if err := jc.DecodeForm("response", &responseFormat); err != nil {
+		return err
```

Comment:
The error returned here is never evaluated, so we probably don't write the proper status code